packages/synapse-react-client/src/components/SynapseTable/SearchV2.tsx

```diff
@@ -327,4 +331,7 @@ class Search extends React.Component<InternalSearchProps, SearchState> {
   }
 }
 
-export default Search
+export default function Search(props: SearchV2Props) {
+  const { data } = useQueryContext()
```

Comment:
Assume same motivation here, to isolate the context to prep for removal?

Reply:
Yep, Jotai (the new state manager) uses hooks so the change becomes a simple hook swap here.